src/docs/sphinx/advancedExamples/validationStudies/mccWellbore/Example.rst

```diff
@@ -0,0 +1,235 @@
+.. _ExampleMCCWellbore:
+
+
+####################################################
+Modified Cam-Clay Model for Wellbore Problems
+####################################################
+
+
+**Context**
+
+In this benchmark example, the Modified Cam-Clay model (see :ref:`ModifiedCamClay`) is applied to solve for elastoplastic deformation within the vicinity of a vertical wellbore. For the presented example, an analytical solution is employed to verify the accuracy of the numerical results. The resulting model can be used as a base for more complex analysis (e.g., wellbore drilling, fluid injection and storage scenarios). 
+
+
+**Input file**
+
+Everything required is contained within two GEOSX input files located at:
+
+.. code-block:: console
+
+  src/coreComponents/physicsSolvers/solidMechanics/benchmarks/PlasticWell_Base.xml
+  src/coreComponents/physicsSolvers/solidMechanics/benchmarks/ModifiedCamClay_Verification.xml
+
+
+------------------------------------------------------------------
+Description of the case
+------------------------------------------------------------------
+
+We simulate a drained wellbore problem subjected to isotropic horizontal stress (:math:`\sigma_h`) and vertical stress (:math:`\sigma_v`), as shown below. By increasing the wellbore supporting pressure (:math:`P_w`), the wellbore expands, and the formation rock experiences elastoplastic deformation. A plastic zone develops in the near wellbore region.
```

Comment:
```suggestion
We simulate an undrained wellbore problem subjected to isotropic horizontal stress (:math:`\sigma_h`) and vertical stress (:math:`\sigma_v`), as shown below. By increasing the wellbore supporting pressure (:math:`P_w`), the wellbore expands, and the formation rock experiences elastoplastic deformation. A plastic zone develops in the near wellbore region.
```

Reply:
For drained case, injecting pressure will cause fluid flow. Only for undrained condition P_w could be regarded as a normal compression to the borehole wall.

Reply:
The drained response corresponds to zero pore pressure or completely dissipated excess pore pressure. In this case, no pore pressure is assumed and only the mechanical deformation is simulated.